core/src/main/scala/io/chrisdavenport/epimetheus/Histogram.scala

```diff
@@ -51,10 +48,10 @@ object Histogram {
    * @param unit The unit of time to observe the timing in. Default Histogram buckets
    *  are optimized for `SECONDS`.
    */
-  def timed[E, F[_]: Bracket[?[_], E]: Clock, A](h: Histogram[F], fa: F[A], unit: TimeUnit): F[A] =
-    Bracket[F, E].bracket(Clock[F].monotonic(unit))
-    {_: Long => fa}
-    {start: Long => Clock[F].monotonic(unit).flatMap(now => h.observe((now - start).toDouble))}
+  def timed[E, F[_]: MonadCancel[*[_], E]: Clock, A](h: Histogram[F], fa: F[A], unit: TimeUnit): F[A] =
```

Comment:
Try `fa.timed`, it's in CE syntax: https://github.com/typelevel/cats-effect/pull/1677